spec/features/viewing_chitters_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'viewing chitter messages' do
+  scenario 'peeps are visible onscreen' do
+    Peeps.create(message: 'hi')
+    visit '/'
+    expect(page).to have_content 'hi'
+  end
+end
```

Comment:
Perhaps another feature test could be useful here to check that the user is able to load the homepage, so that they can add a Peep?